tests/test_other.py

```diff
@@ -9519,3 +9519,34 @@ def test_oformat(self):
 
     err = self.expect_fail([EMCC, path_from_root('tests', 'hello_world.c'), '--oformat=foo'])
     self.assertContained("error: invalid output format: `foo` (must be one of ['wasm', 'js', 'mjs', 'html']", err)
+
+  def test_closure_js_version(self):
```

Comment:
I think I need a few more tests here.. and this one is badly named... but this is the test case that fails with the current codebase.